notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<p>What would we like the reader to take from viewing this table - e.g. important columns. Might be good to point out that 4/5 of the observations are FFI's, one is a processed lightcurve. (If I recall, these might have been described in the previous notebook and could be referenced here.) This is somewhat done in the next block but could explicitly reference the table results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>added cell below table to highlight the important parts of this table</p><p> </p>